driver-core/src/main/java/com/datastax/driver/core/ResultSet.java

```diff
@@ -39,7 +34,11 @@
  * <p/>
  * Note that this class is not thread-safe.
  */
-public interface ResultSet extends Iterable<Row> {
+public interface ResultSet extends Fetchable<ResultSet, Row> {
+
+    // redeclared only to make clirr happy
+    @Override
+    Row one();
```

Comment:
That was the only complaint by clirr plugin. All other methods could be pulled up without problems.